docs/proposals/13-automated-seed-management.md

```diff
@@ -224,17 +240,21 @@ metadata:
   name: crazy-botany
   namespace: garden
 spec:
-  replicas: 1
+  replicas: 3
   selector:
     matchLabels:
       foo: bar
+  updateStrategy:
```

Comment:
Questions to reviewers:
* Since only a single update strategy (`RollingUpdate`) is planned, do we need the entire field as specified here? I tend to prefer it this way, since it's more familiar and open for extensions. Any other opinions?
* Do we need a field similar to `podManagementPolicy` in `StatefulSet`? Or to put it in a different way, do you think "parallel" management policy could also be useful in some cases in addition to the "orderedReady" policy described in the document?